xlive/H2MOD/Variants/Infection/Infection.h

```diff
@@ -22,31 +55,20 @@ class Infection : public ICustomGameVariant
 	virtual void OnMapLoad(ExecTime execTime, s_game_options* gameOptions) override;
 	virtual void OnPlayerSpawn(ExecTime execTime, datum playerIdx) override;
 	virtual void OnPlayerDeath(ExecTime execTime, datum playerIdx) override;
-	
-	// unused bellow
-	virtual void OnObjectDamage(ExecTime execTime, datum unitDatumIdx, int a2, bool a3, bool a4) override
-	{
-	}
 
-	virtual bool OnAutoPickupHandler(ExecTime execTime, datum playerIdx, datum objectIdx) override
-	{
-		return false;
-	}
+	// unused below
+	virtual void OnObjectDamage(ExecTime execTime, datum unitDatumIdx, int a2, bool a3, bool a4) override {}
+	virtual bool OnAutoPickupHandler(ExecTime execTime, datum playerIdx, datum objectIdx) override { return false; }
+	virtual bool OnPlayerScore(ExecTime execTime, void* thisptr, unsigned short a2, int a3, int a4, int a5, char a6) override { return false; }
 
-	virtual bool OnPlayerScore(ExecTime execTime, void* thisptr, unsigned short a2, int a3, int a4, int a5, char a6) override
-	{
-		return false;
-	}
-
-	Infection();
-	~Infection();
```

Comment:
deconstructor should be virtual here too
otherwise good catch

Reply:
https://github.com/pnill/cartographer/pull/407/commits/15ddad208effb18819e21dedf764dd9bbc93430a#diff-722acf67b979a805dd7e77189f0f0f4ee9e23aff90c6145b6c05ab85371695b1R65
Changed it to virtual